index.html

```diff
@@ -30,5 +30,94 @@ <h1 class="text-primary-1 ff-crete line-height-1 ">Hey There. <br> I'm Tomi</h1>
            <li> <a href="#"  class="btn-contact linkedin"> </a> </li> 
         </ul>
     </main>
+    <section class="container work-section grid bg-light">
+        <h2 class="ff-crete mg-8 center">My Recent Works</h2>
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_69](https://user-images.githubusercontent.com/69638013/165134671-c6ef0255-e2e5-4cbc-a5ff-02193fc90091.png)  | ![Screenshot_70](https://user-images.githubusercontent.com/69638013/165134614-71bd3cb5-64f2-4990-bcc0-423a28e6ea12.png)

- [ ] Kindly adjust the spacing between the my recent work header, please reduce the spacing and make it look mere like the Figma design